scripts/fastsp_script/fsp_test.py

```diff
@@ -5,7 +5,8 @@
 ###remember sys.argv - can create variables using indexes with sys.argv!      
 """
 
-seq_dir = "perturbed_alignments_Drosophila_AA/EMGT00050000000018.Drosophila.001_AA"
+#seq_dir = "perturbed_alignments_Drosophila_AA/EMGT00050000000018.Drosophila.001_AA"
+overall_path = sys.argv[1]
```

Comment:
Something to note - this is NOT a function. argv is a _list_ that comes with the sys module. Print out `sys.argv` to prove to yourself. If no parentheses (e.g. `sys.argv()`), not a function.